spring-cloud-gcp-kms/src/test/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.cloud.bootstrap.BootstrapConfiguration=\
```

Comment:
What's the purpose of registering it as a bootstrap configuration?

Reply:
Removed - saw it on `spring-cloud-gcp-secretmanager` and thought it was convention.